data/_models/shared/Organisations.csv

```diff
@@ -1,15 +1,15 @@
 tableName,tableExtends,columnName,columnType,key,required,refSchema,refTable,refLink,refBack,validation,semantics,description,profiles,visible
 Organisations,,,,,,,,,,,"org:Organization,foaf:Agent",Research departments and research groups,"DataCatalogueFlat,EMA",
 Organisations,,overview,heading,,,,,,,,,General information,DataCatalogueFlat,
-Organisations,,id,,1,true,,,,,,dcterms:identifier,Internal identifier,"DataCatalogueFlat,EMA",
-Organisations,,pid,,2,,,,,,,dcterms:identifier,Persistent identifier,"DataCatalogueFlat,EMA",
-Organisations,,acronym,,,,,,,,,dcterms:identifier,Acronym if applicable,"DataCatalogueFlat,EMA",
-Organisations,,name,text,3,true,,,,,,"dcterms:title,foaf:name",Name used in European projects,"DataCatalogueFlat,EMA",
-Organisations,,type,ontology_array,,,CatalogueOntologies,Organisation types,,,,,Type of organisation; in which sector is the organisation active?,"DataCatalogueFlat,EMA",
+Organisations,,id,,1,true,,,,,,,Internal identifier,"DataCatalogueFlat,EMA",
+Organisations,,pid,,2,,,,,,,dct:identifier,Persistent identifier,"DataCatalogueFlat,EMA",
+Organisations,,acronym,,,,,,,,,skos:altLabel,Acronym if applicable,"DataCatalogueFlat,EMA",
+Organisations,,name,text,3,true,,,,,,"dct:title,skos:prefLabel",Name used in European projects,"DataCatalogueFlat,EMA",
+Organisations,,type,ontology_array,,,CatalogueOntologies,Organisation types,,,,org:classification,Type of organisation; in which sector is the organisation active?,"DataCatalogueFlat,EMA",
 Organisations,,type other,text,,,,,,,,,"If type is 'other', a description of type of organisation",DataCatalogueFlat,
-Organisations,,institution,text,,,,,,,,org:FormalOrganization,"University, company, medical centre or research institutes this organisation is part of","DataCatalogueFlat,EMA",
+Organisations,,institution,text,,,,,,,,org:subOrganizationOf,"University, company, medical centre or research institutes this organisation is part of","DataCatalogueFlat,EMA",
```

Comment:
This is outside of DCAT AP scope, should we annotate this?

Reply:
I don't mind. But is not priority. As said before, no problem to annotate to other relevant ontologies. Our goal is to improve semantics of our catalogue, some people use dcat, some people might use other ontologies.